sycl/test/usm/allocator_vector.cpp

```diff
@@ -1,5 +1,6 @@
 // RUN: %clangxx -fsycl %s -o %t1.out -lOpenCL
 // RUN: %CPU_RUN_PLACEHOLDER %t1.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t1.out
```

Comment:
Please put the HOST run before the CPU run to follow the order as in other tests.

Reply:
Done.